Formula/arangodb.rb

```diff
@@ -16,14 +16,33 @@ class Arangodb < Formula
   depends_on :macos => :yosemite
   depends_on "openssl"
 
+  # see https://gcc.gnu.org/bugzilla/show_bug.cgi?id=87665
   fails_with :gcc do
     build 820
     cause "Generates incorrect code"
   end
```

Comment:
> I wanted to express that both gcc 7.3.0 and gcc 8.2.0 have this issues. Is there a better way to do it?

```ruby
fails_with :gcc => "7"
fails_with :gcc => "8"
```

Reply:
@sjackman it does work with 8.3.0, but not with 8.2.0. So I did not want to exclude the whole 8 series.

Reply:
Hmm. As far as I know, the DSL doesn't have a way of specifying that requirement unfortunately. If it does work with 8.3.0, then omit `fails_with :gcc => "8"`.
```ruby
fails_with :gcc => "7"
```

Reply:
@sjackman if have now excluded 820

Reply:
@sjackman ok changed it to 820

Reply:
I don't believe the current code with `build 820` does what you intend.

Reply:
@sjackman ok, changed to `fails_with :gcc => "7"`